components/script/dom/xmlhttprequest.rs

```diff
@@ -1009,14 +1013,14 @@ impl XMLHttpRequest {
                 let upload_complete = &self.upload_complete;
                 if !upload_complete.get() {
                     upload_complete.set(true);
-                    self.dispatch_upload_progress_event(atom!("progress"), None);
+                    self.dispatch_upload_progress_event(atom!("event"), None);
```

Comment:
So it turns out that the spec does not actually want an event name "event". https://xhr.spec.whatwg.org/#request-error-steps shows that it's a variable representing names passed in by code that invokes that algorithm, so this code will require further changes.